libraries/mapr_utils.rb

```diff
@@ -0,0 +1,54 @@
+module Mapr
+  # Module for configure.sh script
+  module ConfigureSh
+    module_function
+
+    MAPR_CONFIGURE_SCRIPT = '/opt/mapr/server/configure.sh'.freeze unless const_defined?(:MAPR_CONFIGURE_SCRIPT)
+
+    # Build the configure.sh command depending on args Hash
+    # args Hash should in a { 'args' => value} format
+    # Example : args = { '-N' => "cluster_name", '-C' => "host1,host2,host3", '-Z' => "host4,host5,host6" }
+    # Options with no value are passed like { '-secure' => true }
```

Comment:
wouldn't `nil` be a better choice for an option without a value?

Reply:
The configure.sh command take options with value like "-C host1,host2,host3" and options with no value, like a flag, to activate or not something, like "-secure", "-R" etc etc

I think it's more logical to explicitly activate or deactivate them with true or false